frontend/elements/src/pages/LoginEmailPage.tsx

```diff
@@ -390,6 +397,22 @@ const LoginEmailPage = (props: Props) => {
               {t("labels.signInPasskey")}
             </Button>
           </Form>
+          {config.providers?.map((provider: string) => {
+            return (
+              <Form key={provider}>
+                <Button
+                  secondary
+                  onClick={(e) => {
```

Comment:
Would prefer to add an `onSubmit`  attribute to the form and remove the `onClick` handler from the button. This way the changes regarding the `<Form>` and `<Button>` components can be reverted.